src/commands/wipe.rs

```diff
@@ -13,17 +14,17 @@ pub(crate) async fn wipe() -> Result<()> {
         }
     };
     let node_directory = node_directory_getter();
-    Node::wipe(node_directory).await?;
+    let _ = Node::wipe(node_directory).await;
```

Comment:
It is better to report errors to users. You can probably save result and after that return a sum of errors of those

Reply:
thanks, I've missed that one.